mesonbuild/environment.py

```diff
@@ -487,6 +487,10 @@ def __init__(self, source_dir: T.Optional[str], build_dir: T.Optional[str], opti
             os.makedirs(self.scratch_dir, exist_ok=True)
             os.makedirs(self.log_dir, exist_ok=True)
             os.makedirs(self.info_dir, exist_ok=True)
+            with open(os.path.join(build_dir, "CACHEDIR.TAG"), mode='w', encoding='utf-8') as cachedir_tag:
+                print("Signature: 8a477f597d28d172789f06886806bc55", file=cachedir_tag)
+                print("# This file is a cache directory tag created by meson.", file=cachedir_tag)
+                print("# For information about cache directory tags, see: http://bford.info/cachedir/", file=cachedir_tag, end='')
```

Comment:
Consistent single quotes, please...

Reply:
Also seems pretty weird to open a file and then print() to it instead of `f.write('contents')`.